Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -181,7 +181,20 @@ export class ResourceGroup {
                         policyLink = this.getPolicyHelpLink(error.details[i]);
                         errorMessage = this.getPolicyErrorMessage(error.details[i]);
                     } else {
-                        errorMessage = util.format("%s: %s %s", error.details[i].code, error.details[i].message, error.details[i].details);
+                        errorMessage = util.format("%s: %s", error.details[i].code, error.details[i].message);
+                        if(error.details[i].details != null && error.details[i].details != undefined) {
+                            if(typeof error.details[i].details == 'object') {
+                                if(Array.isArray(error.details[i].details)) {
```

Comment:
why do we need to handle array specially. Can't we just do `JSON.stringify(error.details[i].details)` ?

Reply:
Wouldn't that print all objects in one single line?

Reply:
Wouldn't that print the whole array as a single string?

Reply:
If you are picking individual details object then you should ideal go down the recursion and pick all child details object on the tree. Because I think `details` can keep on containing details.

JSON.stringify allows us a simpler solution.

Reply:
Makes sense.. I've updated